router/batchrouter/batchrouter.go

```diff
@@ -255,6 +281,49 @@ func (brt *HandleT) Disable() {
 	brt.isEnabled = false
 }
 
+func (brt *HandleT) dedupS3DestJobsOnCrash() {
```

Comment:
Let's remove references to S3 everywhere except in the provider configuration. We shall directly refer to it as objectStorage?